AWSAppSyncClient/AWSAppSyncClient.swift

```diff
@@ -757,6 +829,18 @@ public class AWSAppSyncClient: NetworkConnectionNotification {
                                               resultHandler: resultHandler)
     }
     
+    internal func subscribeWithConnectCallback<Subscription: GraphQLSubscription>(subscription: Subscription, queue: DispatchQueue = DispatchQueue.main, connectCallback: @escaping (() -> Void), resultHandler: @escaping SubscriptionResultHandler<Subscription>) throws -> AWSAppSyncSubscriptionWatcher<Subscription>? {
```

Comment:
Here and throughout: I believe this queue is used to invoke callbacks, right? If so, can we clarify the name to something like `callbackQueue`--right now, the name implies that this is the queue on which the subscription requests and processing will be managed. Either way, is the `main` queue the right default for this? Why wouldn't we create a private queue to be used as a default?

Reply:
Agreed on the naming change. The default is main queue for the assumption that developer would want to do a UI update when the data is available. We can discuss about changing the default queue for this method.